db/schema.rb

```diff
@@ -42,7 +42,7 @@
     t.index ["user_id"], name: "index_api_tokens_on_user_id"
   end
 
-  create_table "audits", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
+  create_table "audits", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
Were all of these changes intentional? IIRC this has happened in the past, but I'm not sure whether it was a good or bad thing.